### PR TITLE
Add more information about Elekto for TOC election

### PR DESCRIPTION
**Pull request by jberkus**
Add more info about Elekto and the voting process to the README, as well as links.

Shorten and make the election page on Elekto more readable.

### CODE REVIEW

**Comment by jberkus** [comment]
tag @bsnchan @thisisnotapril

File: elections/2021-TOC/README.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Please refer to the [TOC Election Charter] for:
 - [Eligibility for candidacy]
 - [Eligibility for voting] 
 
-Eligible Voters list coming soon.
+The list of eligible voters is stored [in the community repository](), but it is easier
```
Comment (bsnchan) [inline elections/2021-TOC/README.md:29]: Are we missing a link here?

**Comment by bsnchan** [comment]
@jberkus Is there a reason why the "Candidacy Process" section is removed?

**Comment by geekygirldawn** [comment]
> @jberkus Is there a reason why the "Candidacy Process" section is removed?

The candidacy process and other details were only removed in the election_desc.md file, which is the one displayed within Elekto that needed to be shortened. All of that detail is still in the readme file in the repo, which is where we should be pointing people to for information about candidacy and other details.

**Comment by bsnchan** [comment]
Got it - thanks for the clarification!

/ok-to-test
/lgtm
/approve

**Pull request merged**